docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -145,6 +155,12 @@ strapi export --no-encrypt # exports your data without encryption.
 # import your data with the default parameters and pass an encryption key: 
 strapi import -f <your-filepath-and-filename> --key my-key
 
+# import your data forcing "yes" for all subsequent prompts:
+strapi import --force
+
+# import only your configuration (schemas are always included): 
+strapi import --only config
```

Comment:
I just realized we don't include -f which is a required option. Is that ok to highlight the options we're giving an example of, or should we provide the full valid command?

Reply:
yes, sorry that needs to be there.